// Ещё одна приятная возможность ES2019 — это то, что аргумент в блоках try-catch теперь стал
//  необязательным. Ранее всем блокам catch надо было передавать, в качестве параметра,
// объект исключения. Аргумент приходилось передавать catch даже в том случае,
// если он не использовался
try {
  let parsed = JSON.parse(obj);
} catch (e) {
  // e можно игнорировать или использовать
  console.log("error");
}

// Теперь это не так. Если объект исключения не используется
// в блоке catch — тогда в этот блок не нужно и ничего передавать.
try {
  let parsed = JSON.parse(obj);
} catch {
  console.log("error");
}
